apps/src/craft/designer/craft.js

```diff
@@ -635,6 +635,7 @@ Craft.runButtonClick = function () {
   if (Craft.level.usePlayer) {
     Craft.showSoftButtons();
   }
+  Craft.gameController.game.input.touch.preventDefault = false;
```

Comment:
does this need to happen on every runbuttonclick or should it be an initialization operation?

Reply:
Ah right, sorry!  I had a different approach before which required the game to be running.  Fixed in 9da3368dd27a3e2f893521d0dacfeb772c049ca5.